tests/console/python3_beautifulsoup4.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: python3-beautifulsoup tests
+# - install python-beautifulsoup package
+# - use library to parse sample htmlfile
+# - Compare the result vs expected
+#
+# Maintainer: QE-Core <qe-core@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils 'zypper_call';
+
+sub run {
+    my $self = shift;
+    select_serial_terminal;
+    # Make sure beautifulsoup python module is installed
+    zypper_call "in python3 python3-beautifulsoup4 python3-lxml";
+    # Import python script and sample html file
+    assert_script_run("curl -O " . data_url("python/bs4/python3-beautifulsoup4-test.py"));
+    assert_script_run("curl -O " . data_url("python/bs4/testpage.html"));
+    # Execute python script. The script itself ensure output is the one expected
+    assert_script_run("python3 python3-beautifulsoup4-test.py");
```

Comment:
Iterate the above steps for all available versions of `beautifulSoup4` in different Python versions